packages/web/README.md

```diff
@@ -44,4 +50,7 @@ This code will make the transaction, let the user sign it using algosigner and s
 
 You can also use `web.sendTransaction()` or `web.signTransaction()` in a react app.
 
+### deployASA and deploySSC
+`deploySSC` has been renamed to `deployASA`.
```

Comment:
you were supposed to update the examples and and text in this readme, rather than adding this note.

Reply:
I have also added an example. Please check.